spec/index.html

```diff
@@ -539,9 +539,7 @@ <h2>RDF Graphs</h2>
 
   <p>An <dfn>RDF graph</dfn> is a set of <a>RDF triples</a>.</p>
 
-  <p class=issue>
-    Do we still need to define <dfn>asserted triple</dfn>?.
-  </p>
+  <p>An RDF triple is said to be <dfn data-lt="asserted|asserted triple">asserted</dfn> in an RDF graph if it is an element of the RDF graph.</p>
```

Comment:
This creates the definition anchor `dfn-asserted`, which seems overly broad. We can tweak the definition so that it creates `dfn-asserted-triple` which may be more suitable.

```suggestion
  <p>An RDF triple is said to be <dfn id="dfn-asserted-triple" data-lt="asserted|asserted triple">asserted</dfn> in an RDF graph if it is an element of the RDF graph.</p>
```